.github/workflows/release.yml

```diff
@@ -19,3 +19,39 @@ jobs:
         env:
           NPM_TOKEN: ${{secrets.npm_token}}
       - run: pnpm publish -r --filter '!@graphql-debugger/landing-page-app' --filter '!@graphql-debugger/landing-page-server' --no-git-checks
+  #TODO
+  #Since we're already building the image in the quality testing, we should try to reuse that.
+  #
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
```

Comment:
why this specific hash?

Reply:
The recommended way of using this action is to lock it to the hash, not sure why not a version but I just picked up the default code, I've taken it from the official docs. Will clean that up eventually.

Reply:
Gotcha, let's go with this then. Do you know what server version they represent? - If so could we add a little comment for it?

Reply:
Yep I'll do that, it's probably the release hash, will add that as a comment